CHANGELOG.md

```diff
@@ -1,123 +1,124 @@
-## 2017-12-8 - Supported Release 2.1.0
+# Changelog
+All notable changes to this project will be documented in this file.
 
-### Summary
-Add support for detecting and installing SQL Server 2017.
-This release does not add any new SQL 2017 specific features.
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-#### Added
+## [Unreleased]
+
+### Added
+- Add CONTRIBUTING.md ([FM-6605](https://tickets.puppetlabs.com/browse/FM-6605))
+
+### Changed
+
+- Update modulesync maintained files.
```

Comment:
Could probably drop this section, changelog is for users, not maintainers (we have the commit logs for that).